manifests/base/grafana-v5/dashboards/driver-extended.yaml

```diff
@@ -1237,20 +1430,90 @@ spec:
             "query": "import \"influxdata/influxdb/schema\"\nschema.tagValues(bucket: \"racing\", tag: \"GameName\")\n",
             "refresh": 1,
             "regex": "",
-            "skipUrlSync": false,
+            "skipUrlSync": true,
             "sort": 1,
             "type": "query"
+          },
+          {
+            "current": {
+              "selected": false,
+              "text": "Racing",
+              "value": "Racing"
+            },
+            "hide": 2,
+            "includeAll": false,
+            "multi": false,
+            "name": "DS_RACING",
+            "options": [],
+            "query": "influxdb",
+            "refresh": 1,
+            "regex": "",
+            "skipUrlSync": true,
+            "type": "datasource"
+          },
+          {
+            "hide": 2,
+            "name": "RegEX",
+            "query": "/^([a-zA-Z0-9ü]+)/",
+            "skipUrlSync": true,
+            "type": "constant"
+          },
+          {
+            "hide": 2,
+            "name": "Domain",
+            "query": "https://pitwall.b4mad.racing",
+            "skipUrlSync": true,
+            "type": "constant"
+          },
+          {
+            "hide": 2,
+            "name": "Extended",
+            "query": "e12c7ccb-ff51-4525-b5da-0d783a1019ef/b4mad-session-details",
+            "skipUrlSync": true,
+            "type": "constant"
+          },
+          {
+            "hide": 2,
+            "name": "Standard",
+            "query": "6d57246de74745198b3385f5e446e81cf7127b47/session",
+            "skipUrlSync": true,
+            "type": "constant"
+          },
+          {
+            "hide": 2,
+            "name": "Driver",
+            "query": "9f21413861900279156aea9e866fdf7adc8629c5/driver",
+            "skipUrlSync": true,
+            "type": "constant"
+          },
+          {
+            "hide": 2,
+            "name": "Track",
+            "query": "c726e9a2-350c-459d-aa40-2035073f5035/track-standings",
+            "skipUrlSync": true,
+            "type": "constant"
           }
         ]
       },
       "time": {
-        "from": "now-7d",
+        "from": "now-2d",
         "to": "now"
       },
-      "timepicker": {},
+      "timepicker": {
+        "hidden": false,
+        "refresh_intervals": [
+          "1m",
+          "5m",
+          "15m",
+          "30m",
+          "1h",
+          "2h",
+          "1d"
+        ]
+      },
       "timezone": "",
-      "title": "Driver Extended",
```

Comment:
lets keep the `UID` and `title` of the dashboard. Otherwise it'll overwrite the normal driver dashboard

Reply:
i added extended to test-it first if not impact the server then we could overwrite